### ISSUE TITLE
[Bug]: Icon in the template footer are not clickable

### ISSUE THREAD

**Issue by alessioventuriniAND** [ONSdigital/design-system#3004]
### What happened?

The icon in the footer are not clickable. This has been highlighted as an accessibility issue in the Dac report of the IDS website. 

### What was the expected outcome?

Icon in the footer to be clickable. 

### How is this problem impacting your service?

accessibility failing 

### What list of steps can be followed to reproduce the bug or issue?

You can look into the IDS live website and when trying to click on the icons in the footer they are not clickable. 

### What version of the ONS Design System are you using?

67.0.4

### What devices are you seeing the problem on?

Desktop, Mobile, Tablet

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge, Microsoft Internet Explorer

**Comment by alessioventuriniAND** [comment]
We need to be able to add links around the icon in the footer.

**Comment by balibirchlee** [comment]
As the custom poweredBy accepts HTML and the IDS website is using this to render a more complicated set up than a simple icon image - it has been agreed that IDS would instead add the links to their existing html. Any other implementation would result in a breaking change, requiring either IDS to simplify their existing footer (i.e. providing only an image) or us to scope an enhancement to the current footer modelled after IDS's current set up.

Whether ourselves or IDS feel that this enhancement should be made is up for discussion, but is beyond the scope of this ticket.

As such, the ticket has been adjusted to focus on providing links for the existing default ONS icons which are  responsive to the language selection (i.e. the Welsh language version which renders the Welsh variant of the default icon has a link to the Welsh site, and vice versa for the English language version).
